lib/manager/swift/extract.ts

```diff
@@ -133,19 +137,19 @@ function getDepName(url: string): string | null {
 }
 
 export function extractPackageFile(
-  content: string,
-  packageFile: string = null
+  extractPackageFileConfig: ExtractPackageFileConfig
```

Comment:
```suggestion
  { fileContent, fileName }: ExtractPackageFileConfig
```

Reply:
@souravdasslg why is this one different?

Reply:
Let me check that